addons/database-cleaner.pl

```diff
@@ -32,16 +32,8 @@ =head1 DESCRIPTION
 use warnings;
 
 use lib '/usr/local/pf/lib';
-use pf::db;
-use DBI;
-use Data::Dumper;
-use Getopt::Long;
-use pf::log;
-use Pod::Usage;
-
 BEGIN {
-  use Log::Log4perl;
-  use pf::log();
+  use Log::Log4perl qw(get_logger);
```

Comment:
The original author wants to bypass the standard pf logging so in this case I am making it explicit that is what is happening.